.circleci/config.yml

```diff
@@ -0,0 +1,214 @@
+version: 2.1
+executors:
+  "gofmt":
+    docker:
+      - image: golang:1.11
```

Comment:
Shouldn't this be 1.12?

Reply:
Ah, that's the version that we run gofmt on; if you compare to our [travis config](https://github.com/stripe/veneur/blob/master/.travis.yml#L13-L18), the versions are the same. Should both be 1.12?

This does have the side-effect of running the source code hygiene checks only on that pinned version of go, which (I think!) might be beneficial; I don't think I've seen a build failure from source code differences that happened because we built protoc-gogofaster with a different version of go, at least (:

Reply:
Yeah, we do want to run gofmt on the most recent version, because they sometimes can change. 

Protobuf output can also change between versions, so it should be the most recent stable version as well.